### PR TITLE
Update rc2 release notes

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
I'd like to keep this as an open PR until the doc is completely finished with all the final details for rc-2.
Please provide feedback and I'll update regularly.

### CODE REVIEW

File: release-notes/SupportedFeatures-v1.0.0-rc2.md [hunk 1]
```diff
@@ -21,38 +29,38 @@ Supported features
...
-|Contracts | DataContract | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | XmlSerializer | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | MessageFormat | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | FaultContract | [:warning:](#contracts_faultcontract) | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-|Security | None | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | Transport | :white_check_mark: | :white_check_mark:  | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
+|Bindings|BasicHttpBinding|:no_entry_sign: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
```
Comment (roncain) [inline release-notes/SupportedFeatures-v1.0.0-rc2.md]: Should all our UWP :no_entry_sign: entries also include links to the section describing why they don't work?

File: release-notes/SupportedFeatures-v1.0.0-rc2.md [hunk 2]
```diff
@@ -141,7 +154,13 @@ Fill in with details.
 
 #### NetTcp  
 &ensp;&ensp; SetLingerOption isn't working correctly on OS X. This is only used on the Close(timeout) code path so the test itself is likely passing. This api required special handling on linux as this is a windows socket concept.
```
Comment (roncain) [inline release-notes/SupportedFeatures-v1.0.0-rc2.md]: This issue with SetLingerOption has been resolved by a fix in CoreFx, so I think we can eliminate this and fix the entries for OS X related to NetTcp

File: release-notes/SupportedFeatures-v1.0.0-rc2.md (hunk 1 above)
Comment (StephenBonikowsky) [inline release-notes/SupportedFeatures-v1.0.0-rc2.md]: Yeah, I can do that.

**Comment by StephenBonikowsky** [comment]
Updated with all known data, updated the column for Centos, had to use Jenkins CI run results for this, no Debian or RedHat runs yet.
@zhenlan merge this at your convenience.

**Comment by zhenlan** [comment]
Thanks @StephenBonikowsky.

**Pull request merged**